src/sass/utils/_variables.scss

```diff
@@ -0,0 +1,14 @@
+:root {
+    --primary-text-color: #131313; // Чорний. Основний колір тексту.
```

Comment:
As we use SASS in our projects, please change the CSS syntax to SCSS syntax in this task. Read   [this](https://www.w3schools.com/sass/sass_variables.php) to learn more.